lib/ui/dashboard/presenter/dashboard_presenter.dart

```diff
@@ -0,0 +1,33 @@
+
+
+import 'package:flutter/material.dart';
+import 'package:rick_and_morty_app/ui/dashboard/interactor/dashboard_interactor.dart';
+import 'package:rick_and_morty_app/ui/dashboard/interactor/dashboard_interactor_provider.dart';
+import 'package:rick_and_morty_app/ui/dashboard/model/list_characters_entity.dart';
+import 'package:rick_and_morty_app/ui/dashboard/presenter/dashboard_listener.dart';
+import 'package:rick_and_morty_app/ui/dashboard/presenter/dashboard_provider.dart';
+
+class DashboardPresenter extends ChangeNotifier implements DashboardProvider, DashboardListener {
+  late DashboardInteractorProvider _interactor;
+  ListCharactersEntity? _listCharacter;
+
+  final listCharactersListenable = ValueNotifier<ListCharactersEntity?>(null);
```

Comment:
O que acha de chamar o método dispose do ValueNotifier? Buscando descartar o objeto quando não for utilizado, liberando o recurso da memória.

Reply:
Resolvido!